engine/src/flutter/impeller/entity/contents/filters/blend_filter_contents.cc

```diff
@@ -840,7 +820,7 @@ std::optional<Entity> BlendFilterContents::CreateFramebufferAdvancedBlend(
       VS::FrameInfo frame_info;
       FS::FragInfo frag_info;
 
-      auto src_sampler_descriptor = SamplerDescriptor{};
+      SamplerDescriptor src_sampler_descriptor = SamplerDescriptor{};
       if (renderer.GetDeviceCapabilities().SupportsDecalSamplerAddressMode()) {
         src_sampler_descriptor.width_address_mode = SamplerAddressMode::kDecal;
```

Comment:
Should this also be removed?

Reply:
Changed this to use the snapshot descriptor, unless we didn't use a snapshot then it just uses a default descriptor.